index.html

```diff
@@ -214,40 +214,94 @@
         "http://heycam.github.io/webidl/#idl-USVString"><dfn>USVString</dfn></a></code> are defined
         in [[!WEBIDL]].
       </p>
-      <p>
-        The term <dfn>webapp</dfn> refers to a Web application, i.e. an application implemented
-        using Web technologies, and executing within the context of a Web <a>user agent</a>, e.g. a
-        Web browser or other Web runtime environment.
-      </p>
-      <p>
-        The term <dfn>webapp server</dfn> refers to server-side components of a <a>webapp</a>.
-      </p>
-      <p>
-        The term <dfn>push message</dfn> refers to an indication to a <a>webapp</a> that there is
-        new information for it from the <a>webapp server</a>, all or part of which MAY be contained
-        in the <a>push message</a> itself.
-      </p>
-      <p>
-        The term <dfn>push subscription</dfn> refers to each logical channel aimed at delivering
-        <a title="push message">push messages</a> from an <a>webapp server</a> to a <a>webapp</a>,
-        which is created by a distinct subscription of such <a>webapp</a> via this Push API.
-      </p>
-      <p>
-        The term <dfn>push service</dfn> refers to an overall end-to-end system that allows
-        <a title="webapp server">webapp servers</a> to send <a title="push message">push
-        messages</a> to a <a>webapp</a>.
-      </p>
-      <p>
-        The term <dfn>push server</dfn> refers to the <a>push service</a> access point via which
-        <a title="webapp server">webapp-servers</a> can initiate <a>push message</a> delivery. Push
-        servers typically expose APIs specific to the <a>push service</a>, e.g. for <a>push
-        message</a> delivery initiation.
-      </p>
-      <p>
-        The term <dfn>express permission</dfn> refers to an act by the user, e.g. via user
-        interface or host device platform features, via which the user approves the permission of a
-        <a>webapp</a> to access the Push API.
-      </p>
+    </section>
+    <section>
+      <h2>
+        Concepts
+      </h2>
+      <section>
+        <h2>
+          Application
```

Comment:
The title is application, but the definition talks about "webapp". I understand that you want to move away from "webapp", but let's not make this more inconsistent than it needs to be?